internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -163,7 +163,11 @@ func CurrentComponentVersion(component Component) string {
 }
 
 func CurrentAddonVersion(addon Addon) string {
-	currentKubernetesVersion := Versions[LatestVersion().String()]
+	return AddonVersionForClusterVersion(addon, LatestVersion())
+}
+
+func AddonVersionForClusterVersion(addon Addon, clusterVersion *version.Version) string {
+  currentKubernetesVersion := Versions[clusterVersion.String()]
```

Comment:
These changes can be dropped here in `versions.go`, now you can iterate over addons and use `AddonVersionForClusterVersion`.

Reply:
I've removed the function and use the new one to iterate over addons. Thank you!